### DISCUSSION TITLE
Too many redirects on assignment's invitation link when an assignment was reused from previous classrooms

### DISCUSSION THREAD

**Discussion by jtyska** [community/community#30929] [GitHub Education]
I've created a new classroom ([INE-5404B-22-2] in the organization INE-UFSC) and used the (amazing) feature of reusing all the assignments from a previous classroom ([INE-5404B-22-1] in the organization INE-UFSC) to create the assignments of this new class. The problem is that when the students tried to use the new invitation link, they have got the 'too many redirects' error. This error doesn't happen when a new 'blank' assignment is created.

**Comment by queenofcorgis** [comment]
Thanks for posting to the GitHub Community, @jtyska! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by DoctorPCOD** [comment]
I am having the same problem! I painstakingly copied over all my assignments from last term, and none of them work. Now I have to generate brand new assignments. :(

**Comment by msridhar** [comment]
Same problem here.  Hope this can get fixed soon!

**Comment by jtyska** [comment]
I was so glad that they have implemented this feature of copying previous assignments. But now it has been 6 months since I reported this bug and no staff member gave any attention to this, so I'm kind of hopeless right now.

**Comment by mislavmatijevic** [comment]
Same error here. All it does is 302s from `/assignment-invitations/{uuid}` to `/assignment-invitations/team/{uuid}` and vice versa.
I'm going to try to create a brand new assignment, hopefully it'll enable me to complete my task.
